releasing.md

```diff
@@ -0,0 +1,304 @@
+
+## Requirements
+
+1. GPG Setup
+2. Sonatype Nexus account
+
+#### Requirement - GPG Setup
+
+Have gpg-agent installed, configured and running.
+
+See these instructions from Github for the complete setup: https://help.github.com/articles/signing-commits-with-gpg/
+
+One way to be sure to check if your gpg agent is running correctly is to encrypt and decrypt a file, using the ```gpg``` tool, don't forget to pass the ```--use-agent``` option, you should be prompted with a dialog (depending on your OS asking for your pass phrase). Now try to decrypt again, you should not be asked again for your pass phrase : your agent is working.
+
+```
+echo "Hello" > doc.txt
+gpg --output doc.gpg --encrypt --recipient <your-gpg-email-address> doc.txt
+gpg --output doc2.txt --decrypt doc.gpg
+```
+
+Contents of doc.txt and doc2.txt should be the same.
+
+Submit your PGP key to https://pgp.mit.edu/ . This is required by Sonatype Nexus!
+
+Find your key and get the armored key:
+```
+$  gpg --list-keys
+
+   /Users/ali/.gnupg/pubring.gpg
+   -----------------------------
+   pub   4096R/10CAEE00 2017-09-25
+   uid       [ultimate] Ali Ok (Ali's GPG key) <aliok@apache.org>
+   sub   4096R/3E608DB2 2017-09-25
+
+$  gpg --armor --export 10CAEE00
+
+    -----BEGIN PGP PUBLIC KEY BLOCK-----
+    Version: GnuPG v2
+
+    mQINBFnI6gsBEAC43ZrS75PAbQCQpvOm70oTMwOPbKaNWqCQLxDR/ck7whBpO9wo
+    ...
+    2rS1FJUwRsqi
+    =CoyC
+    -----END PGP PUBLIC KEY BLOCK-----
+```
+
+Copy the block (including the `--------BEGIN...` and `-----END...` lines) and paste it to https://pgp.mit.edu/ and submit.
+
+#### Requirement - Sonatype Nexus account
+
+If you would like to have a fast sync with Maven central, Sonatype's OSS Nexus is the way to go.
+
+1. Create Sonatype account here: https://issues.sonatype.org/secure/Signup!default.jspa
+2. Check if login works to Nexus: https://oss.sonatype.org/
+3. Ask for publishing permissions for the `groupId` (in pom.xml) of the project you want to publish. Ask @passos or @matzew to add a comment to one of these tickets:
+   * `groupId org.aerogear`: https://issues.sonatype.org/browse/OSSRH-34060
+   * `groupId org.jboss.aerogear`: https://issues.sonatype.org/browse/OSSRH-20598
+
+## Setup
+
+1. GPG
+2. Maven Security
+3. Sonatype Nexus
+
+#### Setup - GPG
+
+Make sure GPG works as described in the requirements section.
+Add your gpg secret key id to your global git config : ```git config --global user.signingkey YOURKEYID```
+
+#### Setup - Maven Security
+
+Set up the password encryption for Maven, you can follow these instructions : http://maven.apache.org/guides/mini/guide-encryption.html
+
+Make sure you created a master password.
+
+### Setup - Sonatype Nexus
+
+First of all, make sure your pom.xml is conforming the requirements of Sonatype: http://central.sonatype.org/pages/apache-maven.html
+
+In order to publish to Sonatype Nexus, have this section in your pom.xml file:
```

Comment:
_in your_ pom ? this is a doc for `digger-java`, right ?  So I'd guess it's that pom

Reply:
yep, right, it is confusing... I started like a more general document and then converted to a Digger Java client specific one :)